dadi/lib/model/revisionManager.js

```diff
@@ -0,0 +1,5 @@
+const Revisions = function (name) {
```

Comment:
is this used anywhere?

Reply:
It isn't! That wasn't supposed to be committed. Thanks!